drivers/net/bnxt/bnxt_hwrm.c

```diff
@@ -1036,6 +1036,10 @@ int bnxt_hwrm_func_vf_stall(struct bnxt *bp, uint16_t vf, uint8_t on)
 		if (rc)
 			break;
 
+		if (vnic.dflt_ring_grp)
+			bp->pf.vnic_dflt_ring_group[vnic.fw_vnic_id] = vnic.dflt_ring_grp;
+		else
+			vnic.dflt_ring_grp = bp->pf.vnic_dflt_ring_group[vnic.fw_vnic_id];
```

Comment:
Has to set dflt_ring_group to 0 to make bd_stall work, next qcfg will return 0, has to cache here.